app/models/user.rb

```diff
@@ -421,6 +421,22 @@ def relationships
     @relationships ||= get_relationships_array
   end
 
+  def identity_serial
```

Comment:
I wouldn't have anything in the `User` model, since this redis isn't really related to it. It shares attributes, but not really something I think the `User` model should know or care about. Instead, I might consider putting this functionality in a service object class. Like `UserProfileAttributeSerializer`, something like that